MediaBrowser.Providers/Tmdb/TV/TmdbSeasonImageProvider.cs

```diff
@@ -0,0 +1,152 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using MediaBrowser.Common.Net;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Controller.Entities.TV;
+using MediaBrowser.Controller.Providers;
+using MediaBrowser.Model.Dto;
+using MediaBrowser.Model.Entities;
+using MediaBrowser.Model.Globalization;
+using MediaBrowser.Model.IO;
+using MediaBrowser.Model.Providers;
+using MediaBrowser.Model.Serialization;
+using MediaBrowser.Providers.Tmdb.Models.General;
+using MediaBrowser.Providers.Tmdb.Movies;
+
+namespace MediaBrowser.Providers.Tmdb.TV
+{
+    public class TmdbSeasonImageProvider : IRemoteImageProvider, IHasOrder
+    {
+        private readonly IJsonSerializer _jsonSerializer;
+        private readonly IHttpClient _httpClient;
+        private readonly IFileSystem _fileSystem;
+        private readonly ILocalizationManager _localization;
+
+        public TmdbSeasonImageProvider(IJsonSerializer jsonSerializer, IHttpClient httpClient, IFileSystem fileSystem, ILocalizationManager localization)
+        {
+            _jsonSerializer = jsonSerializer;
+            _httpClient = httpClient;
+            _fileSystem = fileSystem;
+            _localization = localization;
+        }
+
+        public int Order => 1;
+
+        public string Name => ProviderName;
+
+        public static string ProviderName => TmdbUtils.ProviderName;
+
+        public Task<HttpResponseInfo> GetImageResponse(string url, CancellationToken cancellationToken)
+        {
+            return _httpClient.GetResponse(new HttpRequestOptions
+            {
+                CancellationToken = cancellationToken,
+                Url = url
+            });
+        }
+
+        public async Task<IEnumerable<RemoteImageInfo>> GetImages(BaseItem item, CancellationToken cancellationToken)
+        {
+            var season = (Season)item;
+            var series = season.Series;
+
+            var seriesId = series != null ? series.GetProviderId(MetadataProviders.Tmdb) : null;
+
+            var list = new List<RemoteImageInfo>();
```

Comment:
This isn’t needed anymore, directly assign the value from the ToList call to a variable

Reply:
Thanks for the guidance! Fixed it